examples/cron/compound-staking-cron/src/lib.rs

```diff
@@ -119,25 +119,25 @@ pub fn query(_deps: Deps, env: Env, msg: QueryMsg) -> StdResult<Binary> {
             plane: "COSMOS".to_string(),
             action: "COSMOS_INVOKE".to_string(),
             address: "".to_string(),
-            msg: to_json_vec(&claim_reward).unwrap(),
+            msg: to_json_vec(&claim_reward).unwrap().to_vec(),
```

Comment:
`msg: to_json_vec(&claim_reward).unwrap().to_vec(),` this can just be `msg: to_json_vec(&claim_reward).unwrap()`